opencti-platform/opencti-graphql/src/modules/internal/csvMapper/csvMapper-domain.ts

```diff
@@ -100,7 +100,7 @@ export const csvMapperSchemaAttributes = async (context: AuthContext, user: Auth
   types.forEach((type) => {
     const attributesDef = schemaAttributesDefinition.getAttributes(type);
     const attributes: CsvMapperSchemaAttribute[] = Array.from(attributesDef.values()).flatMap((attribute) => {
-      if (INTERNAL_ATTRIBUTES.includes(attribute.name)) return [];
+      if (INTERNAL_ATTRIBUTES.includes(attribute.name) || attribute.label === 'Original creation date') return [];
```

Comment:
I don't understand this fix, could you explain a bit more how this works ?

Reply:
the name `created` is used for 2 types of attributes :
- Created (only on Incident response, Request for information, Request for takedown)
- Original creation date (for many of other entities)

We want to display only the first one `created` for these 3 entities, not the second one `Original creation date`

Reply:
I think it's a coincidence that the label of created attribute is not the same for Cases and other entities, and I don't think we should have a test on labels like "Original creation date" that could change.

I don't know yet what we could do. Maybe add a condition on mandatory attributes ? INTERNAL_ATTRIBUTES list is not enough.

Reply:
I have improved the condition. We can discuss to improve it if necessary